ng2-components/ng2-alfresco-datatable/README.md

```diff
@@ -188,14 +191,56 @@ onRowClick(event) {
     (executeRowAction)="onExecuteRowAction($event)"
     (rowClick)="onRowClick($event)"
     (rowDblClick)="onRowDblClick($event)">
-    <no-content-template>
-        <template>
-            <h1>Sorry, no content</h1>
-        </template>
-    </no-content-template>
+    
+    
+        <no-content-template>
+            <!--Add your custom empty template here-->
+            <template>
+                <h1>Sorry, no content</h1>
+            </template>
+        </no-content-template>
+        
 </alfresco-datatable>
 ```
 
+### Loading content template
+
+You can add a template that will be showed during the loading of your data:
+
+```html
+<alfresco-datatable
+    [data]="data"
+    [actions]="contentActions"
+    [multiselect]="multiselect"
+    [loading]=isLoading()"
+    (showRowContextMenu)="onShowRowContextMenu($event)"
+    (showRowActionsMenu)="onShowRowActionsMenu($event)"
+    (executeRowAction)="onExecuteRowAction($event)"
+    (rowClick)="onRowClick($event)"
+    (rowDblClick)="onRowDblClick($event)">
+    
+        <loading-content-template>
+            <ng-template>
+               <!--Add your custom loading template here-->
+                <md-progress-spinner
+                    class="adf-document-list-loading-margin"
+                    [color]="'primary'"
+                    [mode]="'indeterminate'">
+                </md-progress-spinner>
+            </ng-template>
+        </loading-content-template>
+        
+</alfresco-datatable>
+```
+
+```js
+    isLoading(): boolean {
+        //your custom logic to identify if you are in a loading state 
+    }
+```
+
+Note: the ```html <loading-content-template> ``` and ```html <no-content-template> ``` can be used together
```

Comment:
For inline content you can use single backticks without format, i.e. \`\<loading-content-template>\`